[DOCS] Removes the SQL block config guides. (#7886)
great-expectations/great_expectations: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

### 4. Connect to your Data

> The remaining sections in Part 1 contain a simplified description of [how to connect to your data in GCS](/docs/0.15.50/guides/connecting_to_your_data/cloud/gcs/pandas) or [BigQuery](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery) and eventually build a <TechnicalTag tag="checkpoint" text="Checkpoint"/> that will be migrated to Cloud Composer. 

Reviewer: ```suggestion
The remaining sections in Part 1 contain descriptions of [how to connect to your data in Google Cloud Storage (GCS)](/docs/0.15.50/guides/connecting_to_your_data/cloud/gcs/pandas) or [BigQuery](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery) and build a <TechnicalTag tag="checkpoint" text="Checkpoint"/> that you'll migrate to Google Cloud Composer. 
```
Define acronyms on first use.

Are Checkpoints always migrated to Google Cloud Composer when using GCS _and_ BigQuery, or is it only GCS?

In the example, we have created a Datasource named `my_bigquery_datasource`, using the `add_or_update_sql` method and passing in a connection string.

> For more details on how to configure the BigQuery Datasource, please refer to [How to connect to a BigQuery database](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery)

Reviewer: ```suggestion
To configure the BigQuery Datasource, see [How to connect to a BigQuery database](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery).
```
![image](https://github.com/great-expectations/great_expectations/assets/104205257/e05456ea-7737-4bab-a3ee-241b5d140610)

Use _see_ instead of _refer to_.  See [Link text](https://developers.google.com/style/link-text).

```python name="tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_bigquery.py save_expectation_suite"
```

> For more details on how to configure the BatchRequest, as well as an example of how you can load data by specifying a table name, please refer to [How to connect to a BigQuery database](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery)

Reviewer: ```suggestion
To configure the BatchRequest and learn how you can load data by specifying a table name, see [How to connect to a BigQuery database](/docs/0.15.50/guides/connecting_to_your_data/database/bigquery)
```
